corehq/apps/app_manager/commcare_settings.py

```diff
@@ -70,7 +70,11 @@ def _load_commcare_settings_layout(doc_type):
         section['title'] = ugettext_noop(section['title'])
         for i, key in enumerate(section['settings']):
             setting = settings.pop(key)
-            if doc_type == 'Application' or setting['type'] == 'hq':
```

Comment:
> In general I think app manager code would be more readable and less error-prone if logic specific to the app type were separated more cleanly.

Agree with this. I went with your solution https://github.com/dimagi/commcare-hq/pull/21911/commits/2ded6d56bdb7677c30fe609cf8cd76fb7083f516

Though this has slightly different behavior than before if there were ever any apps not included in these three. (I think this is fine, but I'm always hesitant in app manager because it's easy to forget things like care plans and shadow modules and various things like that)